compiler/src/dotty/tools/dotc/core/tasty/PositionPickler.scala

```diff
@@ -19,7 +19,7 @@ class PositionPickler(pickler: TastyPickler, addrOfTree: tpd.Tree => Option[Addr
   import buf._
   import ast.tpd._
 
-  private val remainingAddrs = new java.util.IdentityHashMap[Tree, Iterator[Addr]]
+  private val pickledIndices = new mutable.HashSet[Int]
```

Comment:
Maybe use a bitset

Reply:
Yes, this looks ideal for a BitSet, since it is bounded and fairly dense.

Reply:
Done.